### DISCUSSION TITLE
patch_match compiling failed, will fall back to edge_pad

### DISCUSSION THREAD

**Discussion by TheOriSpecial** [lkwq007/stablediffusion-infinity#74] [Q&A]
I have a problem, after installing conda and packages i faced this problem: 

(sd-inf) C:\Users\TheSpecial>Python C:\\Users\\TheSpecial\\Desktop\\stablediffusion-infinity-master\\app.py
patch_match compiling failed, will fall back to edge_pad
No module named 'fpie' so PhotometricCorrection is disabled
Traceback (most recent call last):
  File "C:\Users\TheSpecial\Desktop\stablediffusion-infinity-master\app.py", line 298, in <module>
    upload_button_js = load_js("upload")
  File "C:\Users\TheSpecial\Desktop\stablediffusion-infinity-master\app.py", line 293, in load_js
    with open(f"./js/{name}.js", "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: './js/upload.js'

Does anyone know how to fix it?
P.S. I'm using Win10 21H2 x64

**Comment by lkwq007** [comment]
Change directory to `C:\Users\TheSpecial\Desktop\stablediffusion-infinity-master` first.

By the way, it seems that you downloaded the source code package rather than cloned the repo. In that case, the submodules might also be missing.

**Reply by TheOriSpecial** [reply]
Hi again! So... I cloned the repo and dont know what to do next. I'm a beginner in this, sorry :( Can u help me?
![Screenshot_1](https://user-images.githubusercontent.com/106772608/195783412-038adcbe-e81e-45aa-9f94-5e5b759cb249.jpg)

**Reply by lkwq007** [reply]
```
cd stablediffusion-infinity
git submodule update --init
```
and then try run `python app.py` with conda environment inside the `stablediffusion-infinity` directory (might not work with MINGW64 shell, you may need to use Command Prompt or PowerShell).
